src/backends/firebird/row-id.cpp

```diff
@@ -10,6 +10,10 @@
 
 using namespace soci;
 
+#if defined(_MSC_VER)
+#pragma warning(disable:4702)
```

Comment:
This ("unreachable code") can be a pretty useful warning, it would be nice to avoid disabling it if possible but rather work around it in the place where it occurs.